packages/turf-kinks/lib/sweepline-intersections-export.ts

```diff
@@ -0,0 +1,7 @@
+// Get around problems with moduleResolution node16 and some older libraries.
+// Manifests as "This expression is not callable ... has no call signatures"
+// https://stackoverflow.com/a/74709714
```

Comment:
Same thing here, it appears to have default function in both the types and the code. Maybe its a weird build step from the package with the code in it?

https://github.com/rowanwins/sweepline-intersections/blob/master/src/main.js
https://github.com/rowanwins/sweepline-intersections/blob/master/types.d.ts